### PR TITLE
Adding kallisto

### PR DESCRIPTION
**Pull request by nrichart**

### CODE REVIEW

File: stacks/syrah/syrah.yaml [hunk 1]
```diff
@@ -154,6 +154,7 @@ serial_packages:
               amd: +rocm ~cuda
     - intel-oneapi-tbb
     - jasper
+    - kallisto +hdf5 ^hdf5 ~mpi
```
Comment (batatasfritas) [inline stacks/syrah/syrah.yaml]: When looking at my spack 0.18 I don't see a `hdf5` variant on kallisto. Are you using a more recent spack?

**Comment by nrichart** [comment]
yes I did a `spack clone` using the spack from syrah hopping it clones the same version

File: stacks/syrah/syrah.yaml (hunk 1 above)
Comment (nrichart) [reply stacks/syrah/syrah.yaml]: yes I did a `spack clone` using the spack from syrah hopping it clones the same version

**Review by batatasfritas — APPROVED** [review]
LGTM

**Pull request merged**